woocommerce/wc-api-dev
# Add data endpoints for getting currencies

This PR adds endpoints for getting a list of supported currencies, along with their name and symbol broken out. Includes tests.

It implements three new endpoints:
* `/wc/v3/data/currencies` which returns a list of all supported currency codes, their currency name, and symbol as separate fields.
* `/wc/v3/data/currencies/$currency` which returns a currency code, name, and symbol for a specific currency.
* `/wc/v3/data/currencies/current` which returns a currency code, name, and symbol, for the currency that is currently being used on the site.

To Test:
* Run `phpunit` and make sure all tests pass.
* Visit `/wc/v3/data/currencies/` using postman or another tool and look at responses.

Fixes #2.
See previous PR/discussion at #8.
